mixer/adapter/dogstatsd/config/config.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package adapter.dogstatsd.config;
+
+import "gogoproto/gogo.proto";
+
+option go_package="config";
+
+message Params {
+    // Address of the dogstatsd server, e.g. localhost:8125
+    string address = 1;
+
+    // prefix to prepend to all metrics
+    string prefix = 2;
```

Comment:
In my case I've added `service_mesh.` as a pefix to make it more clear in datadog where the metrics came from and group them together without adding it to each metric name manually. I don't think it makes sense per metric since you can make the metric name anything you want it to be any way.